src/compiler/debug.ts

```diff
@@ -266,6 +300,21 @@ namespace ts {
                 }
             }
 
+            // attempt to load extended debugging information
+            try {
+                if (sys && sys.require) {
+                    const basePath = getDirectoryPath(resolvePath(sys.getExecutingFilePath()));
+                    const result = sys.require(basePath, "./compiler-debug") as RequireResult<ExtendedDebugModule>;

```

Comment:
Should we toss this into a `try`/`catch`? Or do we trust that `sys.require` has already been implemented with one?

Reply:
There is a `try` 3 lines above this one...

Reply:
Oh, dorp, I was expecting it to only wrap the `require` and didn't even acknowledge the outer one.

Reply:
I don't want there to be any chance of an exception in trying to optionally load a debugging utility resulting in a crash of the process.